server/schedule/operator/step.go

```diff
@@ -391,12 +405,10 @@ type AddLightLearner struct {
 	ToStore, PeerID uint64
 }
 
-// ConfVerChanged returns true if the conf version has been changed by this step
-func (al AddLightLearner) ConfVerChanged(region *core.RegionInfo) bool {
-	if p := region.GetStorePeer(al.ToStore); p != nil {
-		return p.GetId() == al.PeerID
-	}
-	return false
+// ConfVerChanged returns how much the conf version will increase due to this step.
+func (al AddLightLearner) ConfVerChanged(region *core.RegionInfo) uint64 {
+	peer := region.GetStorePeer(al.ToStore)
+	return b2u64(peer.GetId() == al.PeerID)
```

Comment:
It seems sometimes we use `GetId()` and sometimes we use `Id` directly. I suggest unifying the code style here.